lib/grape/validations/params_scope.rb

```diff
@@ -140,9 +140,14 @@ def require_optional_fields(context, opts)
       end
 
       def validate_attributes(attrs, opts, &block)
```

Comment:
... which begs the question, why is `fail_fast` so special? Why does it need to be split at all?